katportalclient/client.py

```diff
@@ -1400,7 +1388,7 @@ def userlogs(self, start_time=None, end_time=None):
                 'user_id': 1,
                 'attachments': [],
                 'tags': '[]',
-                'timestamp': '2017-02-07 08:47:22',
+                'sample_time': '2017-02-07 08:47:22',
```

Comment:
Have userlogs really changed?  Search and replace all seems to have caused some problems.

Reply:
+1 should still be timestamp.